.github/workflows/cibuildwheels.yml

```diff
@@ -21,12 +21,12 @@ jobs:
     strategy:
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        arch: [x86, amd64]
+        arch: [auto, aarch64]
```

Comment:
What `auto` means here?  Why `x86` and `amd64` are gone?

Reply:
Here `auto` takes all the available architecture for the given OS. Hence it will automatically take `x86` and `amd64`. Also `aarch64` is not available under `auto`, hence it is provided separately.

Reply:
Here `auto` takes all the available architecture for the given OS. Hence it will automatically take `x86` and `amd64`. Also `aarch64` is not available under `auto`, hence it is provided separately.

Reply:
Got it.